components/cam/src/physics/crm/rrtmgp/radiation.F90

```diff
@@ -2656,6 +2682,31 @@ end subroutine output_fluxes_lw
 
    !----------------------------------------------------------------------------
 
+   ! Utility function to reorder an array given a new indexing
+   function reordered(array_in, new_indexing) result(array_out)
```

Comment:
I'm wondering if calling this something like "reorder_1D" would be a better name in case you need a 2D version later on?

Reply:
Also, shouldn't something this generic be in a separate "radiation_utils.F90" module?

Reply:
In the event I need a 2d version I'd overload it. Prefer to keep the name as-is for now until we need that, because the interface would stay the same.